pkg/features/kube_features.go

```diff
@@ -249,7 +249,7 @@ const (
 	CRIContainerLogRotation featuregate.Feature = "CRIContainerLogRotation"
 
 	// owner: @krmayankk
-	// beta: v1.14
+	// ga: v1.20
```

Comment:
leave the beta line as-is to make it easy to see the history, and add the GA line